internal/v2/s2av2.go

```diff
@@ -341,3 +343,12 @@ func createStream(ctx context.Context, s2av2Address string, getS2AStream func(ct
 		stream: gRPCStream,
 	}, nil
 }
+
+// GetS2ATimeout returns the timeout enforced on the connection to the S2A service for handshake.
+func GetS2ATimeout() time.Duration {
```

Comment:
Same here, please add separate unit tests for this function. :)

Reply:
done